### ISSUE TITLE
composer install yields regular file "src"

### ISSUE THREAD

**Issue by dnmvisser** [cirrusidentity/simplesamlphp-module-authoauth2#72]
hii
Trying to get things working with SimpleSAMLphp 2. After I do:

```sh
vendor/bin/composer require cirrusidentity/simplesamlphp-module-authoauth2:v4.0.0-alpha.1 --update-no-dev
```

I end up with `modules/authoauth2`, but in there is a regular file `src` which contains the string `lib`.
I guess this was meant to be a symlink?

**Comment by dnmvisser** [comment]
So, if I manually symlink `lib` to `src`, and also `www` to `public`, then i can do a successful auth on google.

**Comment by pradtke** [comment]
@dnmvisser oh, interesting. It was indeed a symlink and the goal was to make it easier to merge any changes for the SSP 1.X version of the module into SSP 2.  I'll make some updates to switch to `src` and `public` and make a new release.

**Comment by pradtke** [comment]
Resolved in `v4.0.0-beta.1`.  @dnmvisser when you have a chance, could you verify?

**Comment by dnmvisser** [comment]
Apologies it took so long @pradtke but i just tried it out and the symlink issue is fixed, so i'm closing this. 
thx!
